api/src/main/resources/moduleApplicationContext.xml

```diff
@@ -11,6 +11,101 @@
+		<property name="transactionManager"><ref bean="transactionManager"/></property>
+		<property name="target">
+			<bean class="org.openmrs.module.pihmalawi.patient.impl.ExtendedPatientServiceImpl">
+				<property name="extendedPatientDAO">
+					<bean class="org.openmrs.module.pihmalawi.db.hibernate.HibernateExtendedPatientPatientDAO">
+						<property name="sessionFactory"><ref bean="sessionFactory"/></property>
+					</bean>
+				</property>
+			</bean>
+		</property>
+		<property name="preInterceptors">
+			<ref bean="serviceInterceptors" />
+		</property>
+		<property name="transactionAttributeSource">
+			<bean class="org.springframework.transaction.annotation.AnnotationTransactionAttributeSource"/>
+		</property>
+	</bean>
+
+	<bean parent="serviceContext">
+		<property name="moduleService">
+			<list merge="true">
+				<value>org.openmrs.module.pihmalawi.patient.ExtendedPatientService</value>
+				<ref bean="extendedPatientService"/>
+			</list>
+		</property>
+	</bean>
+
+	<bean id="yendaNafePatientRegistrationService" class="org.springframework.transaction.interceptor.TransactionProxyFactoryBean">
+		<property name="transactionManager"><ref bean="transactionManager"/></property>
+		<property name="target">
+			<bean class="org.openmrs.module.pihmalawi.patient.impl.YendaNafePatientRegistrationServiceImpl"/>
+		</property>
+		<property name="preInterceptors">
+			<ref bean="serviceInterceptors" />
+		</property>
+		<property name="transactionAttributeSource">
+			<bean class="org.springframework.transaction.annotation.AnnotationTransactionAttributeSource"/>
+		</property>
+	</bean>
+
+	<bean parent="serviceContext">
+		<property name="moduleService">
+			<list merge="true">
+				<value>org.openmrs.module.pihmalawi.patient.YendaNafePatientRegistrationService</value>
+				<ref bean="yendaNafePatientRegistrationService"/>
+			</list>
+		</property>
+	</bean>
+
+	<bean id="yendaNafePatientRegistrationValidator" class="org.springframework.transaction.interceptor.TransactionProxyFactoryBean">
+		<property name="transactionManager"><ref bean="transactionManager"/></property>
+		<property name="target">
+			<bean class="org.openmrs.module.pihmalawi.validator.impl.YendaNafePatientRegistrationValidatorImpl"/>
+		</property>
+		<property name="preInterceptors">
+			<ref bean="serviceInterceptors" />
+		</property>
+		<property name="transactionAttributeSource">
+			<bean class="org.springframework.transaction.annotation.AnnotationTransactionAttributeSource"/>
+		</property>
+	</bean>
+
+	<bean parent="serviceContext">
+		<property name="moduleService">
+			<list merge="true">
+				<value>org.openmrs.module.pihmalawi.validator.YendaNafePatientRegistrationValidator</value>
+				<ref bean="yendaNafePatientRegistrationValidator"/>
+			</list>
+		</property>
+	</bean>
+
+	<bean id="locationUuidHandler" class="org.springframework.transaction.interceptor.TransactionProxyFactoryBean">
```

Comment:
Same here.  Not sure why this needs to be a standalone service vs. just an additoinal service method in the YendaNafePatientRegistrationService?

Reply:
@mseaton Same reasoning as for extendedPatientService service